rosetta/text/vw_helpers.py

```diff
@@ -318,21 +318,18 @@ def _set_probabilities(self, topics, predictions):
         Others can be derived and appear as "properties"
         (in the decorator sense).
         """
-        topic_sums = topics.sum()
-        self.pr_topic = topic_sums / topic_sums.sum()
+        self._lambda_word_sums = topics.sum()
+        self.pr_topic = self._lambda_word_sums / self._lambda_word_sums.sum()
 
+        # tokens & topic
         word_sums = topics.sum(axis=1)
         self.pr_token = word_sums / word_sums.sum()
-        self.pr_topic_token = topics / topics.sum().sum()
+        self.pr_token_topic = topics / self._lambda_word_sums.sum()
+        self.pr_token_topic.index.name = 'token'
```

Comment:
Seems like the index and columns could each be named.  Perhaps it would be easier to name the index and columns of topics and then everything else would follow.